src/compass/utils/geometry_utils.py

```diff
@@ -265,3 +265,163 @@ def get_unit_vector4component_of_interest(los_inc_angle, los_az_angle, comp='enu
         ]
 
     return unit_vec
+
+
+def enu2rgaz(radargrid_ref, orbit, ellipsoid,
+             lon_arr, lat_arr, hgt_arr,
+             e_arr, n_arr, u_arr,
+             geo2rdr_params=None):
+    '''
+    Convert ENU displacement into range / azimuth displacement,
+    based on the idea mentioned in ETAD ATBD, available in the link below:
+    ("https://sentinels.copernicus.eu/documents/247904/4629150/"
+     "ETAD-DLR-DD-0008_Algorithm-Technical-Baseline-Document_2.3.pdf/"
+     "5cb45b43-76dc-8dec-04ef-ca1252ace434?t=1680181574715")
+
+    Algorithm description
+    ---------------------
+    For all lon / lat / height of the array;
+    1. Calculate the ECEF coordinates before applying SET
+    2. Calculate the normal vector of east / north / up directions of the point
+    3. Scale the normal vector in 2 with ENU displacement to
+       get the displacement in ECEF
+    4. Add the vectors calculated in 4 into 1.
+       This will be the ECEF coordinates after applying SET
```

Comment:
shoot my comment here didn't get attached to the other
2. "calculate the normal vector of ..." Should this be `the unit vector`? Normal means perpendicular to something
3. Same here for `normal`. What are we scaling the vector by? should be "transform" or "reproject"?
4. `Add the vectors calculated in 4 into 1.` this is step 4, is it add the vectors in (3)?

Reply:
Good catch. I've updated the description in the docstring.